.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
Nit: `Ex.` -> `e.g.` or `Example`

Reply:
+1

Reply:
+1

Reply:
Updated. Thx Anish.  That was boilerplate text from the official github template